runtime/complex.go

```diff
@@ -95,6 +104,18 @@ func complexNE(f *Frame, v, w *Object) (*Object, *BaseException) {
 	return GetBool(!e).ToObject(), nil
 }
 
+func complexRAdd(f *Frame, v, w *Object) (*Object, *BaseException) {
+	return complexArithmeticOp(f, "__radd__", v, w, complexAddComplex, complexAddFloat)
+}
+
+func complexRSub(f *Frame, v, w *Object) (*Object, *BaseException) {
+	return complexArithmeticOp(f, "__rsub__", v, w, complexRSubComplex, complexRSubFloat)
```

Comment:
Since the complex*SubComplex() functions are only called in one place, I would just define the function inline here:

```
return complexArithmeticOp(f, "__rsub__", v, w, func(v, w complex128) complex128 {
  ...
})
```